pages/App.js

```diff
@@ -13,18 +16,69 @@ export default function List() {
     },
     {
       id: nanoid(),
-      text: "Hafermilch und Obst kaufen",
+      text: "die Welt retten",
+    },
+    {
+      id: nanoid(),
+      text: "Blumen gießen",
     },
   ]);
 
+  function addItem(event) {
```

Comment:
Could be named `handleSubmit` so that it is clear that we are handling a form submit event.